install.rst

```diff
@@ -0,0 +1,106 @@
+
+-  How to configure OMERO.server to work with LDAP, both using Ansible or using manual configuration.
+
+-  How to install apllications for OMERO.web, such as OMERO.figure.
+
+Resources
+---------
+
+Link to www OMERO at Institutions 
+Link to Simon's presentation (?)
+Link to Ansible
+Link to install docs
+Link to requrements
+Link to playbooks
+Link to App list
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
+
+Ansible
+-------
+
+For OMERO installation, we recommend to use the Ansible software suite https://docs.ansible.com/ansible/latest/index.html which enables infrastructure as a code.
+
+Install ansible (at the time of writing, the ansible-core version was 2.11.12)::
+
+    $ pip install ansible
+
+Clone the deployment examples repository::
+
+    $ git clone https://github.com/ome/omero-deployment-examples.git
+    $ cd omero-deployment-examples
+
+Clone the subrepository for the one-node example::
+
+    $ git clone https://github.com/ome/ansible-example-omero-onenode.git
+    $ cd ansible-example-omero-onenode	
+
+Install requrements::
+
+    $ ansible-galaxy install -r requirements.yml
+
+Creat a file ``hosts.yml`` in the same directory and put into it (replacing the YOUR-HOST-NAME variable as appropriate)::
+
+    all:
+      hosts:
+        YOUR-HOST-NAME 
+
+Edit the ``playbook.yml`` file to change the OMERO root password (by default ``ChangeMe``). Then use this playbook to install the OMERO.server and OMERO.web on one node::
+
+    $ ansible-playbook --become -i hosts.yml playbook.yml
+
+Start OMERO.insight or a Command Line Interface (CLI) and log in to ``localhost`` with the username ``root`` and the password you have set in the ``playbook.yml`` above.
+
+Go to ``localhost/4080`` in your browser and log in to OMERO.web with the same credentials you used for OMERO.insight above.
+
+Docker
+------
+
+Even quicker option than ansible is docker, but you have to be comfortable working with containers.
+
+Assuming that the Docker is installed, clone the deployment examples repository (if not already cloned)::
+
+    $ git clone https://github.com/ome/omero-deployment-examples.git
+    $ cd omero-deployment-examples
+    $ git clone https://github.com/ome/docker-example-omero.git
+    $ cd docker-example-omero
+
+Pull the latest versions of the containers and start them::
+
+    $ docker-compose pull
+    $ docker-compose up -d
+    $ docker-compose logs -f
+
+Go to ``0.0.0.0/4080`` in your browser and log in to OMERO.web as ``root`` with password ``omero``.
+
+Start OMERO.insight or a Command Line Interface (CLI) and log in to ``localhost`` with the same credentials.
+
+
+
+You also have the option to follow the server installation steps at https://omero.readthedocs.io/en/stable/sysadmins/unix/server-installation.html or 
+But you will have to install apps and set up the config according to the ansible playbook anyway.
```

Comment:
"according to the ansible playbook" - Will users know what that means? I guess you have to look in the right place in one of the repos listed above, but we must have a better resource for how to install web apps?

Reply:
Thank you, this is very much WiP yet - needs a complete rehaul